core/src/main/java/jenkins/widgets/HistoryPageFilter.java

```diff
@@ -101,85 +105,100 @@ public void setSearchString(@Nonnull String searchString) {
     /**
      * Add build items to the History page.
      *
-     * @param items The items to be added. Assumes the list of items are in descending queue ID order i.e. newest first.
+     * @param runItems The items to be added. Assumes the list of items are in descending queue ID order i.e. newest first.
      */
-    public void add(@Nonnull List<T> items) {
-        if (items.isEmpty()) {
+    public void add(@Nonnull Iterable<T> runItems) {
```

Comment:
I don't think this is a backwards-compatible change and expect it to break plugins that use this method.